src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/RefreshController.java

```diff
@@ -26,19 +31,23 @@
 public class RefreshController {
 
     @Autowired
-    public RefreshController(RefreshOrchestrator refreshOrchestrator) {
+    public RefreshController(RefreshOrchestrator refreshOrchestrator,
+                             JudicialRefreshOrchestrator judicialRefreshOrchestrator) {
         this.refreshOrchestrator = refreshOrchestrator;
+        this.judicialRefreshOrchestrator = judicialRefreshOrchestrator;
     }
 
     RefreshOrchestrator refreshOrchestrator;
 
+    JudicialRefreshOrchestrator judicialRefreshOrchestrator;
+
     @PostMapping(
             path = "/am/role-mapping/refresh",
             produces = V1.MediaType.MAP_ASSIGNMENTS,
             consumes = {"application/json"}
     )
     @ResponseStatus(code = HttpStatus.ACCEPTED)
-    @ApiOperation("refreshes role assignments")
+    //@ApiOperation("refreshes role assignments")
```

Comment:
Response code 202 is not applicable for this endpoint. we can add 200 success code and 422 rejection code.